mpp/shmem-def.h.in

```diff
@@ -84,6 +88,33 @@ typedef struct shmem_impl_ctx_t { int dummy; } * shmem_ctx_t;
 #define SHMEM_CTX_SERIALIZED    (1l<<1)
 #define SHMEM_CTX_NOSTORE       (1l<<2)
 
+/* Teams */
+
+typedef struct shmem_impl_team_t {
+    int dummy;
+} * shmem_team_t;
+
+typedef struct {
+    int num_contexts;
+} shmem_team_config_t;
+
+#if SHMEM_HAVE_ATTRIBUTE_VISIBILITY == 1
+    __attribute__((visibility("default"))) extern shmem_team_t SHMEM_TEAM_WORLD;
+    __attribute__((visibility("default"))) extern shmem_team_t SHMEM_TEAM_SHARED;
+#else
+    extern shmem_team_t SHMEM_TEAM_WORLD;
+    extern shmem_team_t SHMEM_TEAM_SHARED;
+#endif
+
+#define SHMEM_TEAM_INVALID NULL
+
+#define SHMEM_CTX_INVALID NULL
+
+#define SHMEM_TEAM_NUM_CONTEXTS       (1l<<0)
```

Comment:
@davidozog Is this a spec requirement? I am trying to understand whether this is an initial value and may change.

Reply:
@wrrobin The definition of SHMEM_TEAM_NUM_CONTEXTS is indeed a spec requirement.  This is the only constant defined by the v1.5 spec for team creation "masks", so I figured 1 is a reasonable value for it.  We could make it whatever value we want, and it could always change if/when we add more mask parameters.